### ISSUE TITLE
v1.4.0 init container image not found

### ISSUE THREAD

**Issue by snipking** [kubernetes-sigs/alibaba-cloud-csi-driver#1038]
#### What happened:

csi-plugin daemonset fail to start due to image `registry.cn-hangzhou.aliyuncs.com/acs/csi-plugin:v1.26.4-e7e52e9-init` not exist.

#### What you expected to happen:

daemonset up and running

#### How to reproduce it (as minimally and precisely as possible):

apply `deploy/ecs` yaml on empty k8s cluster

#### Anything else we need to know?:

nothing

#### Environment:
- CSI driver version (image tag of csi-plugin container): registry.cn-hangzhou.aliyuncs.com/acs/csi-plugin:v1.26.4-e7e52e9-init
- Deployment method (where you got the YAML files, what modifications you made, etc.): apply `deploy/ecs` yamls
- Kubernetes version (use `kubectl version`): v1.26.9+rke2r1
- Cloud provider or hardware configuration (e.g. Alibaba Cloud ECS instance type): ecs.r6.4xlarge
- OS (e.g: `cat /etc/os-release`): centos 7
- Kernel (e.g. `uname -a`): 3.10.0-1160.6.1.el7.x86_64
- Network plugin and version (if this is a network-related bug): 
- Others:

**Comment by snipking** [comment]
found image address in [v1.4.0 release note](https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver/releases/tag/v1.4.0)

which is:

```
registry.cn-beijing.aliyuncs.com/acs/csi-plugin:v1.30.1-242df8a-aliyun-init
registry.cn-beijing.aliyuncs.com/acs/csi-plugin:v1.30.1-242df8a-aliyun
```

**Comment by huww98** [comment]
Should be fixed by #1033
